### ISSUE TITLE
core number calculation needs to force an int result (python3)

### ISSUE THREAD

**Issue by michalstolarczyk** [databio/pepatac#105]
in `bamSitesToWig.py` call the cores calculation fails to produce an integer when running with python 3 and the pipeline fails every time:

```
bamSitesToWig.py: error: argument -p/--cores: invalid int value: '5.333333333333333'
```

[PEPATAC_log.txt](https://github.com/databio/pepatac/files/3152815/PEPATAC_log.txt)

------
https://github.com/databio/pepatac/blob/68a047d78b02489a646e95291f29a66e9911df4f/pipelines/pepatac.py#L1039

**Comment by nsheff** [comment]
this was fixed in a4ecd1910c8f59afa8344ba471ad820591f7bd76

**Comment by dimkonstanto** [comment]
The code is still "cmd += " -p " + str(max(1, int(pm.cores) * 2/3)) " in the most recent pepatac.py script downloaded by github.
